spec/models/season_spec.rb

```diff
@@ -233,9 +233,11 @@
   end
 
   describe '.all_years' do
-    it 'return array of years' do
+    it 'returns array of years that includes succeeding year only if succeeding year is equal to current year' do
```

Comment:
Just a suggestion: it could make sense to also add a season for the _"current"_ year here - as this is something the query explicitly takes care of. Right now, I could go and change the `<=` to `<` without the tests breaking 😉

Reply:
Also: see my comment at the `all_year` method. You may want to add a spec for a transition season. There are some examples that use the different phases of a season, maybe you can use them for inspiration.

Reply:
Good catch! I will implement this.